<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WifiBot</class>
 <widget class="QMainWindow" name="WifiBot">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>442</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>WifiBot</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="sliderVitesse">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>29</width>
      <height>341</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SizeVerCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>Modifier la vitesse du robot</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="BoutonDirectionnel" name="boutonDroite">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>270</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Aller à droite</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="BoutonDirectionnel" name="boutonBas">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Reculer</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Vitesse du Wifibot</string>
    </property>
    <property name="text">
     <string>Vitesse :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonOptions">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Besoin d'aide ?</string>
    </property>
    <property name="text">
     <string>Options</string>
    </property>
   </widget>
   <widget class="BoutonDirectionnel" name="boutonGauche">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Aller à gauche</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="BoutonDirectionnel" name="boutonStop">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Arrêter tout mouvement et réinitialiser la caméra</string>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="BoutonDirectionnel" name="bouttonHaut">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>210</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Avancer</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="BoutonCamera" name="boutCamGauche">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Caméra à gauche</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="BoutonCamera" name="boutCamBas">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Baisser la caméra</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="BoutonCamera" name="boutCamDroit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Caméra à droite</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="BoutonCamera" name="boutCamHaut">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Lever la caméra</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelVid">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>35</y>
      <width>320</width>
      <height>215</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="BoutonCamera" name="boutonCapture">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Prendre une capture d'écran</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="BoutonCamera" name="boutonSon">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Activer/Désactiver le son</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>sliderVitesse</zorder>
   <zorder>boutonDroite</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>boutonOptions</zorder>
   <zorder>boutonBas</zorder>
   <zorder>boutonGauche</zorder>
   <zorder>boutonStop</zorder>
   <zorder>bouttonHaut</zorder>
   <zorder>boutCamGauche</zorder>
   <zorder>boutCamBas</zorder>
   <zorder>boutCamDroit</zorder>
   <zorder>boutCamHaut</zorder>
   <zorder>labelVid</zorder>
   <zorder>boutonCapture</zorder>
   <zorder>boutonSon</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BoutonDirectionnel</class>
   <extends>QPushButton</extends>
   <header>boutondirectionnel.h</header>
  </customwidget>
  <customwidget>
   <class>BoutonCamera</class>
   <extends>QPushButton</extends>
   <header>boutoncamera.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sliderVitesse</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderVitesse</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
